jitsi-media-transform/src/main/kotlin/org/jitsi/nlj/util/IndexTracker.kt

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.nlj.util
+
+/**
+ * Index tracker inspired by the RFC3711 RTP sequence number index tracker.
+ */
+
+class PictureIdIndexTracker : IntIndexTracker(15)
```

Comment:
Add a doc that this is specifically for VP8/VP9 picture IDs, with a rollover of 0x8000?